azure-pipelines.yml

```diff
@@ -33,13 +33,13 @@ stages:
 
 # Test Dependencies - multiple dependencies that are expected in the build steps below this
       dependencies:
-        - source: '\\nirvana\Measurements\VeriStandAddons\prototype\dependency-test-1'
+        - source: '\\nirvana\Measurements\VeriStandAddons\build_tools_test\dependency-test-1'
```

Comment:
Need to edit the test files themselves to address this as well for passing builds.